.github/workflows/pr-release.yaml

```diff
@@ -6,6 +6,17 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
+      - name: Set up GCS Google Service Account
+        run: echo '${{ secrets.GCS_SA }}' > service_account.json
+
+      - name: Lock
+        uses: databiosphere/github-actions/actions/locker@locker-0.8.0
+        with:
+          operation: lock
+          lock_name: terra-helm-release
+          bucket: broad-dsp-locker-action
+          lock_timeout_ms: '120000'
```

Comment:
Does this mean the lock will release no matter what after 120 seconds or the action will fail after trying to aquire the lock for 120 seconds?

Reply:
It will fail, there is another [optional parameter](https://github.com/DataBiosphere/github-actions/tree/master/actions/locker#inputs), `continue_on_lock_timeout` that can be set to true to make it continue instead of failing.